packages/paste-website/src/pages/components/pagination/index.mdx

```diff
@@ -0,0 +1,560 @@
+} from '@twilio-paste/pagination';
+
+const Component = () => {
+  return (
+    <Pagination label="pagination navigation">
+      <PaginationItems>
+        <PaginationArrow variant="back" label="Go to previous page" visibleLabel="Previous" />
+        <PaginationNumbers pageLabel="Page 5 of 10">
+          <PaginationNumber label="Go to page 1">1</PaginationNumber>
+          <PaginationEllipsis label="Collapsed previous pages" />
+          <PaginationNumber label="Go to page 4">4</PaginationNumber>
+          <PaginationNumber label="Go to page 5" isCurrent>
+            5
+          </PaginationNumber>
+          <PaginationNumber label="Go to page 6">6</PaginationNumber>
+          <PaginationEllipsis label="Collapsed next pages" />
+          <PaginationNumber label="Go to page 10">10</PaginationNumber>
+        </PaginationNumbers>
+        <PaginationArrow variant="forward" label="Go to next page" visibleLabel="Next" />
+      </PaginationItems>
+    </Pagination>
+  );
+};
+```
+
+#### Props
+
+#### Pagination Props
+
+| Prop     | Type        | Description                                        | Default |
+| -------- | ----------- | -------------------------------------------------- | ------- |
+| children | `ReactNode` |                                                    | null    |
+| label    | `string`    | `aria-label` for the pagination navigation element | null    |
+
+#### PaginationItems Props
+
+| Prop     | Type        | Description | Default |
+| -------- | ----------- | ----------- | ------- |
+| children | `ReactNode` |             | null    |
+
+#### PaginationLabel Props
+
+| Prop     | Type        | Description | Default |
+| -------- | ----------- | ----------- | ------- |
+| children | `ReactNode` |             | null    |
+
+#### PaginationArrow Props
+
+| Prop          | Type      | Description                                             | Default  |
+| ------------- | --------- | ------------------------------------------------------- | -------- |
+| as?           | `string`  | `a` / `button`                                          | `button` |
+| disabled?     | `boolean` | Sets `aria-hidden` to be true, and `visibility: hidden` | null     |
+| href?         | `string`  | href used when as `a`                                   | null     |
+| label         | `string`  | `aria-label` text                                       | null     |
+| variant       | `string`  | `back` / `forward`                                      | null     |
+| visibleLabel? | `string`  | Visible text of the button or anchor                    | null     |
+
+#### PaginationNumbers Props
+
+| Prop       | Type        | Description                               | Default |
+| ---------- | ----------- | ----------------------------------------- | ------- |
+| children   | `ReactNode` |                                           | null    |
+| pageLabel? | `string`    | Page label text used in `PaginationLabel` | null    |
+
```

Comment:
Is this section missing the PaginationNumber component with the `as?` prop? Is `as=a` set on PaginationNumber or PaginationNumbers?